graphics/src/lib.rs

```diff
@@ -8,8 +8,8 @@
     html_logo_url = "https://raw.githubusercontent.com/iced-rs/iced/9ab6923e943f784985e9ef9ca28b10278297225d/docs/logo.svg"
 )]
 #![deny(
-    missing_debug_implementations,
-    missing_docs,
+    //missing_debug_implementations,
+    //missing_docs,
```

Comment:
Meant to leave this commented?

Reply:
Nope, couple things left I want to do in this PR still!

Thanks for the heads up.

Reply:
Gotcha, will hold off on final comments, though it's looking great so far! Will definitely make things less awkward for Custom Shader [RFC](https://github.com/iced-rs/rfcs/pull/23) 🥳 Having custom backends be able to define their own primitive types definitely makes the most sense.